File: Start.java
public class Start {
    static String colorSettings;
    static String colorNames;
    static String colorRender;
    static String cgenerate;

    static {
        colorNames = Color.BLUE;
        colorSettings = Color.PURPLE;
        colorRender = Color.CYAN;
        cgenerate = Color.DARKCYAN;
    }

    public static void prints(String[] names, String[] render_names, String[] generate_names) {
        Util.print(
                colorNames +
                        Color.UNDERLINE +
                        "Chats" +
                        Color.END +
                        colorNames +
                        ": " +
                        String.join(", ", names) +
                        Color.END
        );
        Util.print(
                colorRender +
                        Color.UNDERLINE +
                        "Will Render" +
                        Color.END +
                        colorRender +
                        ": " +
                        Color.BOLD +
                        (render_names == Util.ALL ?
                                "All" :
                                (Color.BOLD +
                                        (render_names.length == 0 ?
                                                "None" :
                                                String.join(", ", render_names)
                                        )
                                ) + Color.END
                        )
        );
        Util.print(cgenerate +
                Color.UNDERLINE +
                "Will Generate" +
                Color.END +
                cgenerate +
                ": " +
                (Color.BOLD +
                        (generate_names == Util.ALL ?
                                "All" :
                                Color.BOLD +
                                        (generate_names.length == 0 ?
                                                "None" :
                                                String.join(", ", render_names)
                                        )
                        ) + Color.END
                )
        );

        Util.warnUser(render_names, names);
    }

    public static void settings(boolean renderFinal, boolean optimize) {
        Util.print(colorSettings + Color.BOLD + "Starting with settings:" + Color.END);
        Util.print("    " + colorSettings + Color.UNDERLINE + "render mutual graph(s)" + Color.END +
                colorSettings + ": " + renderFinal);
        Util.print("    " + colorSettings + Color.UNDERLINE + "optimize" + Color.END +
                colorSettings + " (don't generate time-consuming data): " + optimize);
    }

    public static void uncolor() {
        colorNames = "";
        colorSettings = "";
        colorRender = "";
        cgenerate = "";
    }
}

File: start.py
import util
import color

cset = color.PURPLE
cnames = color.BLUE
crender = color.CYAN
cgenerate = color.DARKCYAN


def prints(names, render_names, generate_names):
    print(cnames + color.UNDERLINE + f"Chats" + color.END + cnames + ": " + (', '.join(names)) + color.END)
    print(crender + color.UNDERLINE + f"Will Render" + color.END + crender + ": " +
          (color.BOLD + "All" if render_names == ["*"] else
           color.BOLD + "None" if render_names == [] else ', '.join(render_names)) + color.END)
    print(cgenerate + color.UNDERLINE + f"Will Generate" + color.END + cgenerate + ": " +
          (color.BOLD + "All" if generate_names == ["*"] else
           color.BOLD + "None" if generate_names == [] else ', '.join(generate_names)) + color.END)

    util.warn_user(render_names, names)


def settings(render_final, optimize):
    print(cset + color.BOLD + f"Starting with settings:" + color.END)
    print("    " + cset + color.UNDERLINE + f"render mutual graph(s)" + color.END +
          cset + f": " + str(render_final))
    print("    " + cset + color.UNDERLINE + f"optimize" + color.END +
          cset + f" (don't generate time-consuming data): " + str(optimize))


def uncolor():
    global cset
    global cnames
    global crender
    global cgenerate
    cset = ''
    cnames = ''
    crender = ''
    cgenerate = ''
